products/jbrowse-desktop/src/util.tsx

```diff
@@ -16,19 +15,20 @@ export async function fetchCJS(url: string): Promise<LoadedPlugin> {
     const pluginLocation = path.join(tmpDir, sanitize(url))
     const pluginLocationRelative = path.relative('.', pluginLocation)
 
-    await new Promise((resolve, reject) => {
-      const file = fs.createWriteStream(pluginLocation)
-      http
-        .get(url, res => {
-          res.pipe(file)
-          file.on('finish', resolve)
-        })
-        .on('error', err => {
-          fs.unlinkSync(pluginLocation)
-          reject(err)
-        })
-    })
-    return window.require(pluginLocationRelative) as LoadedPlugin
+    let pluginText = ''
+    try {
+      const pluginResponse = await fetch(url)
+      if (!pluginResponse.ok) {
+        throw new Error('Bad response')
+      }
+      pluginText = await pluginResponse.text()
+    } catch (error) {
+      console.error(error)
+      await fsPromises.unlink(pluginLocation)
```

Comment:
not sure if I already commented this but might be able to remove this unlink, and maybe whole catch block, the pluginLocation I think is not written in any case here so might not need unlink?

Reply:
Good point, that was a leftover from the old `http`-style fetch. I removed the whole try-catch, and improved the error message while I was at it.